### DISCUSSION TITLE
Linux Support

### DISCUSSION THREAD

**Discussion by IceDBorn** [garungorp/MouseInjectorDolphinDuck#4] [Ideas]
Is this even usable with wine?

**Comment by daemonspudguy** [comment]
It's based on the ManyMouse library so it should be possible to port to Linux. How exactly is another story entirely.

**Comment by daemonspudguy** [comment]
@garungorp do you have any ideas?

**Comment by daemonspudguy** [comment]
It's using MinGW I just noticed. So someone with programming skills better than mine (not difficult) could be able to port this to Linux.

**Comment by zerocoregithub** [comment]
With the popularity of the Steam Deck this would be a very welcome addition.

I would love to play games like Nightfire with mouse-look on my Steam Deck using the gyro while also retaining analog movement.

**Comment by Mahim2000** [comment]
Any update on that?

**Comment by Chalking** [comment]
This would be great!!
